### PR TITLE
Release And Bugfixing Strategy

### LINKED ISSUE #328: 📚 Decide and Document a longterm support strategy
As user and developer of the secureCodeBox I would like know the support strategy for this project.
This strategy should include:
- the number of versions currently supported
- EOL date for each version

At this Point there is no offical support timeline for a release.
We currently release about every two weeks dropping support for the previous version.

The strategy will be published in the GitHub repository as well as in the documentation of the secureCodeBox

### PR DESCRIPTION
**Pull request by fuhrmeistery**
We had a discussion about our release management and these are the main
points we want to clarify

Closes #328

Signed-off-by: Yannik Fuhrmeister <yannik.fuhrmeister@protonmail.com>

### CODE REVIEW

**Review by rseedorff — APPROVED** [review]

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+### Minor Release/Feature Releases (Semver)
+We currently plan to provide support for the _latest minor [semver](https://semver.org/lang/de/)_ release only.
+
+### Patch Release/Bugfix/Security Fix
+We try to make bugfixes and high severity fixes available as patch release for the current minor release
+as early as possible.
+
+## Extended (Enterprise) Support
+If you are interessted in extended support for older versions with security updates of our project
```
Comment (twwd) [inline SECURITY.md:27]:
```suggestion
If you are interested in extended support for older versions with security updates of our project 
```

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+### Minor Release/Feature Releases (Semver)
+We currently plan to provide support for the _latest minor [semver](https://semver.org/lang/de/)_ release only.
+
+### Patch Release/Bugfix/Security Fix
+We try to make bugfixes and high severity fixes available as patch release for the current minor release
+as early as possible.
+
+## Extended (Enterprise) Support
+If you are interessted in extended support for older versions with security updates of our project 
+please get in touch with the project team via Slack or eMail <secureCodeBox@iteratec.com>.
```
Comment (twwd) [inline SECURITY.md:28]:
```suggestion
please get in touch with the project team via Slack or email <secureCodeBox@iteratec.com>.
```

File: SECURITY.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+## Reporting a Vulnerability
+You have found a vulnerability in the project that shouldndt be discloused as public issue before its fixed?
+Please get in touch with the project team via Slack or eMail <secureCodeBox@iteratec.com>.
```
Comment (twwd) [inline SECURITY.md:32]:
```suggestion
Please get in touch with the project team via Slack or email <secureCodeBox@iteratec.com>. 
```

File: SECURITY.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+### Patch Release/Bugfix/Security Fix
+We try to make bugfixes and high severity fixes available as patch release for the current minor release
+as early as possible.
+
+## Extended (Enterprise) Support
+If you are interessted in extended support for older versions with security updates of our project 
+please get in touch with the project team via Slack or eMail <secureCodeBox@iteratec.com>.
+
+## Reporting a Vulnerability
+You have found a vulnerability in the project that shouldndt be discloused as public issue before its fixed?
```
Comment (twwd) [inline SECURITY.md:31]:
```suggestion
You have found a vulnerability in the project that shouldn't be disclosed as public issue before it's fixed?
```

File: SECURITY.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+| <= 2.4.x | :x: | :x: |
+| < 2.0 | :x: | :x: |
+
+### Major Release (Semver)
+_Upcoming major updates_ will come with a time window in which both _major versions_ (starting with v2.x.x)
+will receive security updates and bugfixes. The concrete support intervall will be probably a couple of months
+and will be published when the next major version will be released.
+
+### Minor Release/Feature Releases (Semver)
+We currently plan to provide support for the _latest minor [semver](https://semver.org/lang/de/)_ release only.
```
Comment (twwd) [inline SECURITY.md:20]:
```suggestion
We currently plan to provide support for the _latest minor [semver](https://semver.org/)_ release only.
```

File: SECURITY.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
+# Security Policy
+
+## Supported Versions
+Our _release cycle_ for new features (minior [semver](https://semver.org/lang/de/) update)
```
Comment (twwd) [inline SECURITY.md:4]:
```suggestion
Our _release cycle_ for new features (minior [semver](https://semver.org/) update)
```

**Review by twwd — CHANGES_REQUESTED** [review]
I

**Pull request merged**